### ISSUE TITLE
Sync command does not take into account of specific build (based on angular environment files)

### ISSUE THREAD

**Issue by Borongaj** [ionic-team/vscode-extension#71]
I can build, **sync** and run for a specific platform and env from command line (but without live reload).

`ng build --configuration=staging --output-hashing=none && npx cap sync ios && npx cap run ios`

Using the extension (because I need live reload option), I can build production (from build ellipsis menu), but if I click the **sync** button, a new build is made, based on default env file.

Thanks.

**Comment by dtarnawsky** [comment]
Thanks for the bug report Laurent,

This issue has been fixed in 1.12.2 which was just released.

**Comment by Borongaj** [comment]
I still have the same problem with 1.12.2...

Output log :

```
[Ionic] Syncing...
> export ANDROID_HOME=~/Library/Android/sdk && npx ionic cap sync --inline
> ng run app:build
- Generating browser application bundles (phase: setup)...```

**Comment by dtarnawsky** [comment]
Ugh, monday mornings are killing me. I fixed and tested with Cap CLI, pnpm, nx, lerna and yarn but didn't test with a local Ionic CLI. I'll fix that and have 1.12.3 out shortly.....
